--- a/specifications/sdoc-0.3.20.gemspec	2013-08-29 22:43:53.533000000 +0300
+++ b/specifications/sdoc-0.3.20.gemspec	2013-08-29 22:46:22.744000000 +0300
@@ -22,14 +22,14 @@ Gem::Specification.new do |s|
     s.specification_version = 3
 
     if Gem::Version.new(Gem::VERSION) >= Gem::Version.new('1.2.0') then
-      s.add_runtime_dependency(%q<rdoc>, ["~> 3.10"])
+      s.add_runtime_dependency(%q<rdoc>, [">= 3.10"])
       s.add_runtime_dependency(%q<json>, [">= 1.1.3"])
     else
-      s.add_dependency(%q<rdoc>, ["~> 3.10"])
+      s.add_dependency(%q<rdoc>, [">= 3.10"])
       s.add_dependency(%q<json>, [">= 1.1.3"])
     end
   else
-    s.add_dependency(%q<rdoc>, ["~> 3.10"])
+    s.add_dependency(%q<rdoc>, [">= 3.10"])
     s.add_dependency(%q<json>, [">= 1.1.3"])
   end
 end
